### PR TITLE
[PM-9952][PM-9954] Minor assign collection fixes

### PR DESCRIPTION
**Pull request by nick-livefront**
## 🎟️ Tracking

[PM-9952](https://bitwarden.atlassian.net/browse/PM-9952)
[PM-9954](https://bitwarden.atlassian.net/browse/PM-9954)

## 📔 Objective

PM-9954 - Truncate `select` options to use ellipsis overflow and have a `title` so hovering with a mouse will display the option name
PM-9952 - Disable org selector when there is only a single organization

## 📸 Screenshots

|Long Org Name|Disabled Selector|
|-|-|
|<video src="https://github.com/user-attachments/assets/0d0d9549-f071-4733-bf19-d377a1f4bf7e"/>|<video src="https://github.com/user-attachments/assets/a4db9b57-4cc8-49b4-ba9a-3af0d7ec9524" />|

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

[PM-9952]: https://bitwarden.atlassian.net/browse/PM-9952?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[PM-9954]: https://bitwarden.atlassian.net/browse/PM-9954?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/components/src/select/select.component.html
Comment (nick-livefront) [inline libs/components/src/select/select.component.html:1]: @vleague2 - These were called out as a part of [PM-9954](https://bitwarden.atlassian.net/browse/PM-9954) and felt like minor changes. Let me know if you have any concerns with scenarios that I'm not thinking of!

**Review by Jingo88 — APPROVED** [review]

**Comment by gbubemismith** [comment]
Changes look good 👍🏼
❓ Would there be a separate PR to address the wording Danielle pointed out?

**Review by gbubemismith — APPROVED** [review]

**Comment by nick-livefront** [comment]
> Changes look good 👍🏼 ❓ Would there be a separate PR to address the wording Danielle pointed out?

@gbubemismith Yes! Those will be addressed in [PM-9951](https://bitwarden.atlassian.net/browse/PM-9951)

**Review by vleague2 — APPROVED** [review]
Component changes are small and align with design, so I am good with it!

**Pull request merged**